backend/tests/factories.py

```diff
@@ -10,3 +10,20 @@ class Meta:
     author = factory.Sequence(lambda n: 'Simon %s' % n)
     code = 'print(123)'
     is_example = False
+
+
+class ConfigFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Config
```

Comment:
Nie ma zaimportowanego Config, ktoś tu nie testował przed :D

Reply:
testował tylko nie na tym samym urządzeniu co wrzucał, ponieważ docker i windows się nie chcą dogadać, zapominałam o linijce przy przerzucaniu kodu, więc jak coś to testy działają, tylko nie wszystko przerzuciłam